### ISSUE TITLE
video extention

### ISSUE THREAD

**Issue by HaoZhang990127** [THUDM/CogVideo#357]
### Feature request  / 功能建议

Hi,

the readme mention video continuation, is it use i2v to extend the video generated by t2v? Do you have any other suggestions for video extention?
<img width="930" alt="b77ea983a344ac3ae788b6a964347118" src="https://github.com/user-attachments/assets/c351cb30-f8d6-4b27-ba78-0dd31fa4526c">

### Motivation / 动机

video extention

### Your contribution / 您的贡献

video extention

**Comment by zRzRzRzRzRzRzR** [comment]
This job is not at the model level, you can check the implementation of `CogVideoXVideoToVideoPipeline` in diffusers

**Comment by HaoZhang990127** [comment]
Thank you for your reply. As I understand CogVideoXVideoToVideoPipeline is used for that writing on the video style, it doesn't serve to extend the video duration, if I want to get a 12s or longer video, do you have any suggestions? Is there something wrong with my understanding? Thank you for your reply.

**Comment by Neethan54** [comment]
Hi @HaoZhang990127 ,

Currently , All the models from  CogvideoX can generate maximum 6 seconds video, with fps=8.

**Comment by HaoZhang990127** [comment]
ok, thank you for your reply.

**Comment by Neethan54** [comment]
How much time it is taking for you to generate 6 second video?

**Comment by zRzRzRzRzRzRzR** [comment]
Yes, this is actually generating the next 6-second video, for each individual video, the duration does not exceed 6 seconds

**Comment by zRzRzRzRzRzRzR** [comment]
Yes, this is actually generating the next 6-second video, for each individual video, the duration does not exceed 6 seconds

**Comment by HaoZhang990127** [comment]
> @HaoZhang990127 How much time it is taking for you to generate 6 second video?

around 5 min in h800
